### DISCUSSION TITLE
Is it possible to set the alt-text of an image sent as a MessageAttachment?

### DISCUSSION THREAD

**Discussion by Ethazeriel** [discordjs/discord.js#7128] [Q&A] [answered]
As per title - I've tried a few things but haven't had any success. [This](https://github.com/discord/discord-api-docs/discussions/3295) seems to suggest I need to add a description field to my attachment (also [this](https://discord.com/developers/docs/resources/channel#attachment-object)), although none of  the ways I've tried to do that have worked - [this](https://discord.js.org/#/docs/main/stable/class/MessageAttachment) seems to suggest it's possible to send arbitrary data as a third argument, but it's unclear quite how to format that and nothing I've tried has worked. Would appreciate some guidance here :)

**Comment by Jiralite** [comment, marked as answer]
discord.js has not yet released this to stable. If you want to use it now, you will have to use the main branch (`npm install discord.js@dev`) which means you'll be able to [set descriptions of attachments](https://discord.js.org/#/docs/main/main/class/MessageAttachment?scrollTo=setDescription). Otherwise, you'll have to wait for a release.

**Reply by Ethazeriel** [reply]
makes sense - thanks for the prompt reply!
